### DISCUSSION TITLE
Network error: Response not successful: Received status code 413

### DISCUSSION THREAD

**Discussion by Kurayami-Senpai** [requarks/wiki#6827] [Help / Questions] [answered]
I found the article https://github.com/Requarks/wiki/issues/2797
It would basically answering the question. However, I can't find the master.js file in which I can adapt the changes. I've been looking for it for a few hours now and still can't find it anywhere.

My system:

Operating System:     Ubuntu 22.04.3 LTS
Wiki:                           Docker Container (Linux) | Wiki.js version 2.5.300
DB:                             PostgreSQL 15.4
Note.js:                       18.17.0

**Comment by NGPixel** [comment, marked as answer]
Set the `bodyParserLimit` property in your config.yml instead.

https://github.com/requarks/wiki/blob/main/config.sample.yml#L147

**Reply by Kurayami-Senpai** [reply]
That's the problem in my Wiki.js root directory /etc/wiki there is nothing except the .db secret.
I can't find any config.yml
Do I have to create this myself if I have installed Wiki.js via Docker or should it actually be found there ?
